ramls/request.json

```diff
@@ -324,6 +324,10 @@
         }
       }
     },
+    "destinationItemId": {
```

Comment:
Related to my comment above, I think introducing a property that isn't mean to be used most of the time is giving us some strong design feedback about this API.

Reply:
I am for putting the destination item id in the path. What do you propose for the this and the request body? Currently requires a request type. There would be more code involved building the initial request and related objects wrapper.